WooCommerce/UITestsFoundation/Screens/Products/ProductFilterScreen.swift

```diff
@@ -0,0 +1,32 @@
+import ScreenObject
+import XCTest
+
+public final class ProductFilterScreen: ScreenObject {
+
+    private let stockStatusButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["Stock Status"]
+    }
+
+    private let showProductsButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.staticTexts["Show Products"]
+    }
+
+    private var stockStatusButton: XCUIElement { stockStatusButtonGetter(app) }
+    private var showProductsButton: XCUIElement { showProductsButtonGetter(app) }
+
+
+    public init(app: XCUIApplication = XCUIApplication()) throws {
+        try super.init(
+            expectedElementGetters: [ stockStatusButtonGetter, showProductsButtonGetter ],
+            app: app
+        )
+    }
+
+    public func setStockStatusFilterAs(filter: String) throws -> ProductsScreen {
```

Comment:
Taking the function name into account, explicitly mentioning the argument name looks a bit redundant, and can be shortened with:

```suggestion
    public func setStockStatusFilterAs(_ filter: String) throws -> ProductsScreen {
```

Then the call looks like:

```swift
.setStockStatusFilterAs("Out of stock")
```

Reply:
agreed that it's redundant and good tip! updated in https://github.com/woocommerce/woocommerce-ios/pull/9312/commits/b00549daee683cc6d3af5e6c4a03eb75e691efef